### ISSUE TITLE
Enhance a11y for `SegmentedControl`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1264]
path: stories/components/forms/segmented-control.stories.tsx

### Description
1 accessibility violation was found in the `SegmentedControl`.

#### [isReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-forms-segmentedcontrol--is-read-only)

Expected the HTML found at $('#\:r3o\:') to have no violations:
    <div id=":r3o:" aria-readonly="true" class="ui-segmented-control css-1mymdiu">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('.css-1j77xsm[value="孫悟空"][data-checked=""]') to have no violations:
    <label value="孫悟空" aria-readonly="true" data-checked="" class="ui-segmented-control__button css-1j77xsm">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3o\:-0') to have no violations:
    <input disabled="" readonly="" id=":r3o:-0" type="radio" name="segmented-control-:r3p:" aria-readonly="true" data-checked="" class="css-0" value="孫悟空" checked="" data-index="0" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3o\: > .css-1j77xsm[value="ベジータ"]') to have no violations:
    <label value="ベジータ" aria-readonly="true" class="ui-segmented-control__button css-1j77xsm">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3o\:-1') to have no violations:
    <input disabled="" readonly="" id=":r3o:-1" type="radio" name="segmented-control-:r3p:" aria-readonly="true" class="css-0" value="ベジータ" data-index="1" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('.css-1j77xsm[value="フリーザ"]') to have no violations:
    <label value="フリーザ" aria-readonly="true" class="ui-segmented-control__button css-1j77xsm">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3o\:-2') to have no violations:
    <input disabled="" readonly="" id=":r3o:-2" type="radio" name="segmented-control-:r3p:" aria-readonly="true" class="css-0" value="フリーザ" data-index="2" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3q\: > .css-1j77xsm[value="ベジータ"]') to have no violations:
    <label value="ベジータ" aria-readonly="true" class="ui-segmented-control__button css-1j77xsm">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

Expected the HTML found at $('#\:r3q\:-1') to have no violations:
    <input disabled="" readonly="" id=":r3q:-1" type="radio" name="segmented-control-:r3r:" aria-readonly="true" class="css-0" value="ベジータ" data-index="1" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;">

    Received:
    Elements must only use supported ARIA attributes (aria-allowed-attr)

    Fix all of the following:
      ARIA attribute is not allowed: aria-readonly="true"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/aria-allowed-attr?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
